django-verdant/rca/static/rca/css/streamfield-blocks/video-block.less

```diff
@@ -0,0 +1,55 @@
+.block-video {
+    position: relative;
+
+    &__title {
+        position: absolute;
+        top: 13px;
+        left: 15px;
+        z-index: 3;
+        color: @color-white !important;
+        margin: 0;
+        // Reduce width to force line-breaks
+        max-width: 140px;
+
+        @media @media-desktop-small {
+            line-height: 0.75;
+        }
+
+        .playing & {
+            display: none;
```

Comment:
Ha! that was easier than I was expecting... thought we'd need to listen to a js event